docs/getting_started.md

```diff
@@ -203,6 +203,136 @@ See these resources for more information on clustering:
+
+
+#### `SingleR` annotations
+
+`SingleR` annotation uses a reference dataset from the [`celldex` package](https://bioconductor.org/packages/release/data/experiment/html/celldex.html) [[Aran _et al._ (2019)](https://doi.org/10.1038/s41590-018-0276-y)].
+
+
+To access automated `SingleR` annotations as cell type names and/or ontology terms in the process `SingleCellExperiment` object, use the following command(s):
+
+```r
+# SingleR annotatins as cell type names
+processed_sce$singler_celltype_annotation
+
+# Or, SingleR annotatins as cell type ontology terms
+processed_sce$singler_celltype_ontology
+```
+
+To access automated `SingleR` annotations as cell type names and/or ontology terms in the processed `AnnData` object, use the following command(s):
+
+```r
+# SingleR annotatins as cell type names
+processed_adata.obs["singler_celltype_annotation"]
+
+# Or, SingleR annotatins as cell type ontology terms
+processed_adata.obs["singler_celltype_ontology"]
+```
+
+Cells that `SingleR` could not confidently annotate are labeled with `NA`.
+
+You can also access the full object returned by `SingleR` from the `SingleCellExperiment`'s metadata with the following command (note that this information is not provided in the `AnnData` object):
+
+```r
+# SingleR full result
+metadata(processed_sce)$singler_results
+```
+
+Additional information about `SingleR` annotation, including the reference source and version, is also available from the processed `SingleCellExperiment` object's metadata and from the processed `AnnData` object's `uns` slot, as described in the {ref}`experiment metadata table<sce_file_contents:SingleCellExperiment experiment metadata>`.
+
+
+#### `CellAssign` annotations
+
+`CellAssign` annotation uses a reference set of marker genes from the [`PanglaoDB` database](https://panglaodb.se/) [[Oscar Franzén _et al._ (2019)](https://doi.org/10.1093/database/baz046)], as compiled by the Data Lab for a given tissue group.
+
+To access automated `CellAssign` annotations in the `SingleCellExperiment`, use the following command:
+
+```r
+# CellAssign annotations for each cell
+processed_sce$cellassign_celltype_annotation
+```
+
+Additional information about `CellAssign` annotation results are also available from the `SingleCellExperiment`'s metadata, as described in the {ref}`experiment metadata table<sce_file_contents:SingleCellExperiment experiment metadata>`.
+
+To access automated `CellAssign` annotations in the `AnnData` object, use the following command:
+
+```python
+# CellAssign annotations for each cell
+processed_adata.obs["cellassign_celltype_annotation"]
+```
+
+Cells that `CellAssign` could not confidently annotate are labeled with `"other"`.
+
+You can also access the full predictions matrix returned by `CellAssign` from the `SingleCellExperiment`'s metadata with the following command:
+
+```r
```

Comment:
add in the python command for this?

Reply:
Oh thanks for pointing this out, I had meant to ask about it. Are we sure we are saving this in AnnData? I don't see it in an object I was working with (which I generated last week) - 

```
>>> adata
AnnData object with n_obs × n_vars = 5047 × 60319
    obs: 'barcodes', 'submitter_celltype_annotation', 'sum', 'detected', 'subsets_mito_sum', 'subsets_mito_detected', 'subsets_mito_percent', 'total', 'prob_compromised', 'miQC_pass', 'scpca_filter', 'sizeFactor', 'cluster', 'singler_celltype_ontology', 'singler_celltype_annotation', 'cellassign_celltype_annotation', 'cellassign_max_prediction', 'library_id', 'sample_id', 'scpca_project_id', 'submitter_id', 'participant_id', 'submitter', 'age', 'sex', 'diagnosis', 'subdiagnosis', 'tissue_location', 'disease_timing', 'organism', 'development_stage_ontology_term_id', 'sex_ontology_term_id', 'organism_ontology_id', 'self_reported_ethnicity_ontology_term_id', 'disease_ontology_term_id', 'tissue_ontology_term_id', 'assay_ontology_term_id', 'suspension_type', 'is_primary_data'
    var: 'gene_ids', 'gene_symbol', 'mean', 'detected', 'feature_is_filtered'
    uns: 'X_name', 'af_num_cells', 'af_permit_type', 'af_resolution', 'alevinfry_version', 'assay_ontology_term_id', 'cellassign_reference', 'celltype_methods', 'cluster_algorithm', 'cluster_nn', 'cluster_weighting', 'filtering_method', 'highly_variable_genes', 'include_unspliced', 'library_id', 'mapped_reads', 'mapping_tool', 'min_gene_cutoff', 'normalization', 'prob_compromised_cutoff', 'project_id', 'reference_index', 'salmon_version', 'sample_id', 'schema_version', 'scpca_filter_method', 'seq_unit', 'singler_reference', 'singler_reference_label', 'tech_version', 'total_reads', 'transcript_type', 'usa_mode'
    obsm: 'X_PCA', 'X_UMAP'
    layers: 'logcounts', 'spliced'
```

But this seems to be the only place we removed content from the metadata - 

https://github.com/AlexsLemonade/scpca-nf/blob/263b6e7d94ae119323fd706efe5d1f157c577419/bin/sce_to_anndata.R#L89-L90

Reply:
Ah you are right. I think we are removing it in the scpcaTools function. https://github.com/AlexsLemonade/scpcaTools/blob/bc767a55dbc0a702b89c73e6d6b5a8a45ba986f5/R/sce_to_anndata.R#L54

Reply:
Ah you are right. I think we are removing it in the scpcaTools function. https://github.com/AlexsLemonade/scpcaTools/blob/bc767a55dbc0a702b89c73e6d6b5a8a45ba986f5/R/sce_to_anndata.R#L54